addons/hr_recruitment/views/hr_job_views.xml

```diff
@@ -116,4 +116,24 @@
             </kanban>
         </field>
     </record>
+
+    <!-- hr related job position menu action -->
+    <record model="ir.actions.act_window" id="action_hr_job">
```

Comment:
Two kanban views for hr.job.
Wich one to use for action_hr_job_config ?

Reply:
again, the other kanban view is dashboard of hr_recruitment, so i have defined the view_id in dashboard's action (action_hr_job), the other opens from config menu (by defualt the list view). so view_id is not needed for kanban view its working fine.